src/common/copilot/dataverseMetadata.ts

```diff
@@ -44,6 +45,32 @@ export async function getEntityColumns(entityName: string, orgUrl: string, apiTo
     }
 }
 
+export async function getFormXml(entityName: string, formName: string,  orgUrl: string, apiToken: string, telemetry: ITelemetry, sessionID: string): Promise<(string [])>{
+    try {
+        const dataverseURL = `${orgUrl.endsWith('/') ? orgUrl : orgUrl.concat('/')}api/data/v9.2/systemforms?$filter=objecttypecode eq '${entityName}' and name eq '${formName}'`;
```

Comment:
let's move this to constants. and can we use `URL method` to create URL.

Reply:
Whole URL can't be moved to constants as it contains dynamic values. Moved the` apiPath` to constants.